student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(age, name, parent_permission, classroom)
+    super(age, name, parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] In this method, please kindly make the default name to "unknown" and parent_permission default to "true".

Like this:

```ruby
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
end
```